src/ape/managers/project.py

```diff
@@ -434,16 +437,17 @@ def compiler_data(self) -> List[Compiler]:
 
         return compilers
 
-    # @property
-    # def meta(self) -> PackageMeta:
-    #     return PackageMeta(**self.config.get_config("ethpm").serialize())
-
-    # def publish_manifest(self):
-    #     manifest = self.manifest.to_dict()  # noqa: F841
-    #     if not manifest["name"]:
-    #         raise ProjectError("Need name to release manifest")
-    #     if not manifest["version"]:
-    #         raise ProjectError("Need version to release manifest")
-    #     # TODO: Clean up manifest and minify it
-    #     # TODO: Publish sources to IPFS and replace with CIDs
-    #     # TODO: Publish to IPFS
+
+# @property
```

Comment:
I don't think this indentation change is needed but at the same time I really wish we did not have commented out code in the code base

Reply:
Black has now created the issue of needing the extra line :(

Reply:
Misunderstood and I've reverted it